clang/lib/Sema/SemaOverload.cpp

```diff
@@ -156,7 +156,8 @@ ImplicitConversionRank clang::GetConversionRank(ImplicitConversionKind Kind) {
                      // it was omitted by the patch that added
                      // ICK_Zero_Queue_Conversion
     ICR_C_Conversion,
-    ICR_C_Conversion_Extension
+    ICR_C_Conversion_Extension,
+    ICR_C_Conversion_Extension,
```

Comment:
What's the consequence of using `ICR_C_Conversion_Extension` here instead of just an ordinary conversion?  In a sense, this isn't an extension in the same way that some of the other conversions are — these are just fundamental types that C++ hasn't incorporated into the standard yet.

Reply:
`ICR_C_Conversion_Extension` I picked pretty much because of the `Extension` bit. Functionally, using `ICR_Conversion` doesn't change anything since none of the other implicit conversion kinds that have a rank from `ICR_Conversion` to `ICR_C_Conversion_Extension` should be applicable for fixed point types. I think block check where we assign `ICK_Fixed_Point_Conversion` should make sure we don't add any other SCSs.